stroom-core/src/main/java/stroom/core/query/SuggestionsQueryHandler.java

```diff
@@ -0,0 +1,10 @@
+package stroom.core.query;
```

Comment:
Move this iface to stroom.util.filter in stroom-util.  We don't want other modules to have to pull in stroom-core as we are slowly trying to get rid of it.

Reply:
Moved to new module: `stroom.query.impl`.